metron-interface/metron-alerts/cypress/integration/pcap/pcap.spec.js

```diff
@@ -0,0 +1,228 @@
+/// <reference types="Cypress" />
```

Comment:
Can this line be removed?

Reply:
This turns on cypress specific intellisense in VSCode and some other editors. I have removed it at first but code completion etc. are way more intelligent with this setup.
You can read more about it here https://docs.cypress.io/guides/tooling/intelligent-code-completion.html#Triple-slash-directives